ui-templates/custom-types/meta_data.json

```diff
@@ -3,37 +3,37 @@
         "python::module": "simulariumio",
         "python::object": "MetaData",
         "parameters": {
-            "box_size": {
+            "size": {
                 "name": "Box size",
                 "data_type": "coordinates_xyz",
                 "description": "Provide the size of the bounding box in XYZ",
                 "required": false,
                 "help": "If not provided, defaults to [100, 100, 100]"
             },
-            "camera_defaults": {
+            "cameraDefault": {
```

Comment:
intentional change to remove the trailing `s` ?

Reply:
good eye, but yes that was intentional. simularium file format just has `cameraDefault`, no `s`